servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/DelegatingExecutor.java

```diff
@@ -37,67 +35,67 @@ public class DelegatingExecutor extends DelegatingListenableAsyncCloseable imple
      */
     protected DelegatingExecutor(final Executor delegate) {
         super(delegate);
-        this.delegate = delegate;
     }
 
     /**
-     * Returns the delegate {@link Executor} used.
+     * Get the {@link Executor} that this class delegates to.
      *
-     * @return The delegate {@link Executor} used.
+     * @return the {@link Executor} that this class delegates to.
      */
     @Override
+    @SuppressWarnings("PMD.UselessOverridingMethod") // Method is overridden to preserve binary compatibility
     protected Executor delegate() {
-        return delegate;
+        return super.delegate();
```

Comment:
I had to keep `delegate()` overrides for binary compatibility. Idk why jampcmp is mad if I remove this method. It makes no sense to me, taking into account that the original method inside `DelegatingAsyncCloseable` is not final, and we did the same trick in #3241